test/callback_api.js

```diff
@@ -57,10 +57,18 @@ function waitForMessages(ch, q, k) {
 
 suite('connect', function() {
 
-test('at all', function(done) {
-  connect(doneCallback(done));
+  test('at all', function(done) {
+    connect(doneCallback(done));
+  });
+
 });
 
+suite('updateSecret', function() {
+  test('updateSecret', function(done) {
+    connect(kCallback(function(c) {
+      c.updateSecret(Buffer.from('new secret'), 'no reason', doneCallback(done));
+    }));
+  });
```

Comment:
Tests the callback version of updateSecret. Not a great test as there are no side effects to assert. I checked that the operation and reply are sent/received via WireShark though.